signals/subscription_all_persons.py

```diff
@@ -0,0 +1,36 @@
+import swapper
+
+from django.dispatch import receiver
+
+from rest_framework import status
+from rest_framework.response import Response
+
+from django.db.models.signals import (
+    post_save
+)
+
+from categories.models import (
+    UserSubscription,
+    Category
+)
+
+@receiver(post_save, sender = Category)
+def subscription_all_persons(sender, instance, **kwargs):
+    
+    """ Database signal receiver for the notification subscription of all Persons
+    to every new Category created.
+    :param sender:
+    :param instance:
+    :param kwargs:
+    :return:
+    """
+    
+    persons_list = swapper.load_model('kernel', 'Person').objects.all()
+    persons_list = list(persons_list)
+    
+    for person in persons_list:
```

Comment:
This could be pretty slow in production with thousands of person objects. Try to come up with some optimizations for this
https://stackoverflow.com/questions/12681653/when-to-use-or-not-use-iterator-in-the-django-orm This might help